src/buffalonwb/add_behavior.py

```diff
@@ -14,7 +18,8 @@ def add_behavior(nwbfile, behavior_file, metadata_behavior):
     )
 
     # Player Position
-    pos = Position(name='Position')
+    meta_pos = metadata_behavior['Position']
+    pos = Position(name=meta_pos['name'])
```

Comment:
we should assign description here if defined. Would `pos = Position(**meta_pos)` work?

Reply:
[Position](https://pynwb.readthedocs.io/en/stable/pynwb.behavior.html#pynwb.behavior.Position) does not have description.
`pos = Position(**meta_pos)` would not work because what's on `spatial_series` field is metadata about the spatial_series to be constructed, not SpatialSeries objects